src/index.js

```diff
@@ -0,0 +1,149 @@
+// WAVE 2
+const increaseTemp = () => {
+  const increaseButton = document.getElementById('increase-temp');
+
+  increaseButton.addEventListener('click', () => {
+    const currentTemperature = document.getElementById('curr-temp');
+    currentTemperature.innerHTML = parseInt(currentTemperature.innerHTML) + 1;
+    changeTempColorAndLandscape();
+  });
+};
+
+const decreaseTemp = () => {
+  const decreaseButton = document.getElementById('decrease-temp');
+
+  decreaseButton.addEventListener('click', () => {
+    const currentTemperature = document.getElementById('curr-temp');
+    currentTemperature.innerHTML = parseInt(currentTemperature.innerHTML) - 1;
+    changeTempColorAndLandscape();
```

Comment:
I recommend refactoring the increaseButton listener and the decreaseButton listener to DRY the code - one possibility would be to create a helper function that does this same work and uses a parameter to determine how much to change the temperature.